packages/fxa-auth-server/package.json

```diff
@@ -39,8 +39,8 @@
     "audit-orphaned-stripe-accounts": "CONFIG_FILES='config/secrets.json' node -r esbuild-register ./scripts/audit-orphaned-customers.ts",
     "remove-unverified-accounts": "CONFIG_FILES='config/secrets.json' node -r esbuild-register ./scripts/remove-unverified-accounts.ts",
     "emails-scss": "node -r esbuild-register ./lib/senders/emails/sass-compile-files.ts",
-    "storybook": "npm run install-ejs && yarn emails-scss && NODE_OPTIONS=--openssl-legacy-provider start-storybook -p 6010 --no-version-updates -s ./",
-    "build-storybook": "npm run install-ejs && yarn emails-scss && NODE_OPTIONS=--openssl-legacy-provider build-storybook && yarn build-storybook:copy-locales && yarn build-storybook:copy-templates",
+    "storybook": "yarn l10n-prime && npm run install-ejs && yarn emails-scss && NODE_OPTIONS=--openssl-legacy-provider start-storybook -p 6010 --no-version-updates -s ./",
```

Comment:
nit: some of these are `yarn <script>` and others are `npm run <script>`.
Not sure if it matters, or we should convert all to `yarn` for consistency (not in this PR, but guess I should file a separate issue).

```sh
git grep "npm run " packages/*/package.json | wc -l # 22
```

Reply:
Thanks! This has been bugging me for a while, but I didn't want to change something that was out of scope.